packages/common/src/components/Preference/PreferenceNumber.tsx

```diff
@@ -1,12 +1,13 @@
-import React, { ChangeEvent, FunctionComponent, ComponentProps } from 'react';
+import React, { ChangeEvent, FunctionComponent } from 'react';
 
 import { Input } from './elements';
 
 type Props = {
   setValue: (value: number) => void;
   step?: number;
   value: number;
-} & Pick<ComponentProps<typeof Input>, 'style'>;
+  style?: React.CSSProperties;
```

Comment:
@siddharthkp This should be reverted (see @SaraVieira's PR)

Reply:
okay